### DISCUSSION TITLE
Ollama Chat Completion GetStreamingChatMessageContentsAsync not invoking plugin

### DISCUSSION THREAD

**Discussion by eeadie** [microsoft/semantic-kernel#9956] [Q&A]
Hi,

I have an issue when using AddOllamaChatCompletion with GetStreamingChatMessageContentsAsync where it is not invoking plugins and is returning an empty result. If I replace GetStreamingChatMessageContentsAsync with GetChatMessageContentAsync the plugin is invoked as expected and a result is returned.

Note both work as expected when using AddAzureOpenAIChatCompletion. Please see attached example showing the issue:

[SemanticKernelLightBlub.zip](https://github.com/user-attachments/files/18112962/SemanticKernelLightBlub.zip)

**Comment by moonbox3** [comment]
Tagging @markwallace-microsoft for visibility.

**Comment by markwallace-microsoft** [comment]
Created this issue https://github.com/microsoft/semantic-kernel/issues/9988
